### PR TITLE
Update hpc_training_rules.adoc

### PR DESCRIPTION
**Pull request by memani1**
Added text to state the training rules version applicable at the time of HPC rules freeze.

### CODE REVIEW

**Comment by memani1** [comment]
updated in https://github.com/mlcommons/training_policies/commit/6fba576cd23385763831eda9556c17ef2a0b3a2e

File: hpc_training_rules.adoc [hunk 1]
```diff
@@ -4,15 +4,16 @@
...
-July 16, 2020
+Version 2.0: 
+August 22, 2022
 
 Points of contact: David Kanter (dkanter@gmail.com), Steve Farrell (sfarrell@lbl.gov)
 
 == Overview
-
-All rules are taken from the MLPerf Training Rules except for those that are overridden here.
+2
```
Comment (sparticlesteve) [inline hpc_training_rules.adoc]: stray character?

File: hpc_training_rules.adoc [hunk 2]
```diff
@@ -4,15 +4,16 @@
...
 
 = MLPerf HPC Training Rules
+<!-- Version 0.7  July 16, 2020 -->
 
-Version 0.7 
-July 16, 2020
+Version 2.0: 
+August 22, 2022
 
 Points of contact: David Kanter (dkanter@gmail.com), Steve Farrell (sfarrell@lbl.gov)
```
Comment (sparticlesteve) [inline hpc_training_rules.adoc]: reminder that you should add yourself here. That can come later, though.

File: hpc_training_rules.adoc (hunk 1 above)
Comment (memani1) [reply hpc_training_rules.adoc]: ah yes, fixed it 6f9afb56ba373e212dbe5d93282b82ac4d4c4b4e

**Review by sparticlesteve — DISMISSED** [review]

**Comment by sparticlesteve** [comment]
Just needs a merge for the conflicts.

**Comment by sparticlesteve** [comment]
I fixed the conflict.

**Review by sparticlesteve — APPROVED** [review]

**Pull request merged**